desktop/apps/article2/components/__tests__/App.test.js

```diff
@@ -3,29 +3,35 @@ import components from '@artsy/reaction-force/dist/components/publishing/index'
 import fixtures from 'desktop/test/helpers/fixtures.coffee'
 import React from 'react'
```

Comment:
Not for this PR, but do you think it makes sense to export our reaction fixtures so that we can use identical data for testing across repos?